db/migrate/20160510173147_create_peer_reviews.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePeerReviews < ActiveRecord::Migration
+  def change
+    create_table :peer_reviews do |t|
+      t.references :result, index: true, foreign_key: true, null: false
+      t.integer :reviewer_id, null: false
+      t.integer :reviewee_id, null: false
```

Comment:
We decided at the meeting to remove this column (it is redundant, or more precisely a functional dependency with result_id). **But** you can leave the association in the model there, just make it a `:through` something.